juniper_official/routing/check-isis-flap-detection.rule

```diff
@@ -54,6 +54,14 @@
                 }
                 type integer;
                 description "Isis neighbor flap count";
+                plots flap-plots {
```

Comment:
remove "-plots" and give more appropriate name

Reply:
changed to neighbor-flaps